data/benefits/aide-nationale-au-brevet-daptitude-aux-fonctions-danimateur-bafa.yml

```
@@ -0,0 +1,34 @@
+label: Aide nationale au Brevet d'aptitude aux fonctions d'animateur (Bafa)
```

```suggestion
label: aide nationale au Brevet d'aptitude aux fonctions d'animateur (Bafa)
```
C'est un peut technique mais le label doit commencer par une minuscule dans le cas où ce n'est pas un nom propre, la majuscule du début de phrase sera ajoutée automatiquement.